### PR TITLE
feat: Adding ability to create a subscription at HEAD

### PR DESCRIPTION
**Pull request by hannahrogers-google**

### CODE REVIEW

File: google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/AdminClient.java [hunk 1]
```diff
@@ -108,6 +119,17 @@ static AdminClient create(AdminClientSettings settings) throws ApiException {
    */
   ApiFuture<Subscription> createSubscription(Subscription subscription);
```
Comment (dpcollins-google) [inline google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/AdminClient.java:120]: `default ApiFuture<Subscription> createSubscription(Subscription subscription) { createSubscription(subscription, OffsetLocation.END); }`

Comment (dpcollins-google) [reply google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/AdminClient.java:120]: Also modify the comment to make it clear where this creates the subscription

**Review by dpcollins-google — CHANGES_REQUESTED** [review]
LGTM after fixes. Ping me once fixed

File: google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/AdminClient.java (hunk 1 above)
Comment (hannahrogers-google) [reply google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/AdminClient.java]: Done. This is ready for another look. Thanks!

**Review by dpcollins-google — APPROVED** [review]

**Pull request merged**